src/Microsoft.Extensions.Configuration.Abstractions/project.json

```diff
@@ -6,7 +6,8 @@
     "nowarn": [ "CS1591" ],
     "xmlDoc": true
   },
-  "description": "Abstractions of key-value pair based configuration.",
+  "description": "Abstractions of key-value pair based configuration.\r\nCommonly used types:\r\nMicrosoft.Extensions.Configuration.IConfiguration\r\nMicrosoft.Extensions.Configuration.IConfigurationBuilder\r\nMicrosoft.Extensions.Configuration.IConfigurationRoot\r\nMicrosoft.Extensions.Configuration.IConfigurationProvider",
```

Comment:
Include `IConfigurationSection` among commonly used types?  I get that the list doesn't need to be complete, but depending of the point of view,  you end up using this one more than plain `IConfiguration`.